manifests/master/puppetdb_conf.pp

```diff
@@ -30,9 +31,16 @@
       value   => $port,
     }
   } else {
-    ini_setting { 'puppetdbserver_urls':
-      setting => 'server_urls',
-      value   => "https://${server}:${port}/",
+    unless $servers {
+      ini_setting { 'puppetdbserver_urls':
+        setting => 'server_urls',
+        value   => "https://${server}:${port}/",
+      }
+    } else {
+      ini_setting {'puppetdbserver_urls':
+        setting => 'server_urls',
+        value   => join($servers, ',')
+      }
```

Comment:
I mean sorting seems like it would enforce an order you may or may not want.I don't know for sure. But I would guess that puppet does them in the order you put them in. Maybe someone might want them in a specific order?